test/README.md

```diff
@@ -2,7 +2,7 @@
 
 This tooling configures object storage, can enable SSL, creates a default user and workspace, and can set up a VCS for a freshly provisioned CD4PE VMs. It also creates the root account and sets a trial license.
 
-Use of this script requires a local copy of the `.cdpe-workflow-tests-config.json` file, which you can get from 1Password, and save it in your ${HOME} directory.
+Use of the `configTestVm.sh` script requires nothing but Docker. Your `~/.ssh/id_rsa` SSH key will be used to clone dependencies from private GitHub repositories.
```

Comment:
> Your `~/.ssh/id_rsa` SSH key will be used to clone dependencies from private GitHub repositories.

Looking at the code below, I don't think this is strictly true is it? You're mounting in the users `~/.ssh` dir, including their `~/.ssh/config` file - so it will use whatever is setup there, right?

(I have a very messy ssh config to allow me to clone as different github users, different SSH keys depending on who i'm talking to etc, and don't really use id_rsa)

If my assumption is wrong, what would it take to allow us to pass in the ssh key to use?

Reply:
It is true, here: https://github.com/puppetlabs/puppetlabs-cd4pe/pull/80/files#diff-3c32df669beac8326518a5604a06c8e7R9

I wasn't expecting this behavior, but once I got far enough to call the rake litmus task it started downloading all these additional dependencies that didn't seem to be captured by the Gemfile? Oh well.

But it downloaded like 20 private repos, and almost every one was private.  It needs access either via HTTPS username/password which you have to enter in every time (so like ~20 times), or with SSH and an ssh-agent which only requires one prompt.

We could easily make this configurable. It would probably be an ENV variable?

Something like `SSH_KEY=~/.ssh/id_foobar ./configTestVm.sh ...` perhaps?

Reply:
Added an ENV variable for specifying the key: https://github.com/puppetlabs/puppetlabs-cd4pe/pull/80/commits/46aeebcdd54f600cb27b69e68aeca8c1ad69bb69